README.md

```diff
@@ -2,16 +2,40 @@
 
 [![Build Status](https://travis-ci.org/hmrc/api-subscription-fields.svg)](https://travis-ci.org/hmrc/api-subscription-fields) [ ![Download](https://api.bintray.com/packages/hmrc/releases/api-subscription-fields/images/download.svg) ](https://bintray.com/hmrc/releases/api-subscription-fields/_latestVersion)
 
-This microservice stores definitions and values for the HMRC Developer Hub.
+This microservice stores and retrieves subscription fields and their values.
 
+### Fields Definition Endpoint examples
 
-### Tests
+#### Add or modify fields definition record
+
+
+#### Get fields definition by api context and api version
+
+
+
+
+### Subscription Fields Endpoint examples
+
+#### Get subscription fields by fields id
+curl -v -X GET -H "Content-Type: application/json"  -H "Cache-Control: no-cache" "http://localhost:9000/fieldsId/f121ffa3-df94-43a0-8235-ac4530f9700a"
+
+#### Get subscription fields by application id, api context, api version 
+curl -v -X GET -H "Content-Type: application/json"  -H "Cache-Control: no-cache" "http://localhost:9000/application/327d9145-4965-4d28-a2c5-39dedee50334/context/ciao-api/version/1.0"
+
+#### Add or modify a subscription fields record
```

Comment:
Can you move this endpoint on the top of the list?
I suggest to have the endpoints list in this order: first the PUT, then the two GETs, then the DELETE.

Reply:
Done.